pair.py

```diff
@@ -0,0 +1,63 @@
+import itertools
+
+import ortho
+
+
+def projections_work(left_locations, right_locations, left, right, session):
+    for left_location, right_location in zip(left_locations, right_locations):
+        if not (ortho.name_at_location(right, right_location) in session.project_forward(ortho.name_at_location(left, left_location))):
+            return False
+    return True
```

Comment:
```suggestion
    return all(
        ortho.name_at_location(right, right_location) in session.project_forward(ortho.name_at_location(left, left_location))
        for left_location, right_location in zip(left_locations, right_locations)
    )
```

Reply:
I'd also make a function out of the predicate for readability, possibly a nested one if it's unnecessary elsewhere.